applications/views.py

```diff
@@ -158,7 +158,14 @@ class ReviewSubmissionViewSet(
     authentication_classes = (SessionAuthentication,)
     serializer_class = SubmissionReviewSerializer
     permission_classes = (IsAdminUser,)
-    queryset = ApplicationStepSubmission.objects.all()
+    queryset = (
+        ApplicationStepSubmission.objects.all()
```

Comment:
If submissions with `status=pending` should be filtered out, this is the place to do it

Reply:
Sounds good, but I'll leave it to a separate issue to fix this